locales/fr.lua

```diff
@@ -629,32 +630,31 @@ for key,value in pairs({
 		["10 Player (Heroic)"] = "10J (H)",	--TODO: recheck this key
 		["25 Player"] = "25J",	--TODO: recheck this key
 		["25 Player (Heroic)"] = "25J (H)",	--TODO: recheck this key
-		--TODO: ["Emissary Quests"] = "Emissary",
-		--TODO: [TRACKER_HEADER_WORLD_QUESTS] = "WQ",	-- ["World Quests"] = "WQ"
+		["Emissary Quests"] = "Émissaire de quêtes",
+		[TRACKER_HEADER_WORLD_QUESTS] = "WQ",	-- ["World Quests"] = "WQ"
```

Comment:
In french LFR is LFR :P
World Quest is WQ too (because QdM it's not very beautiful)

But *Emissary Quests* is *Émissaire de quête*

Reply:
Again, this table should not have any English at all. The left side should be in French, and the right side as well.

Reply:
Yes I understood, I was just telling you that ^^

Reply:
It's not only English, it can be done in any language. In your example with _Émissaire de quête_ it could be shortened to _Émissaire_ (if it's appropriate, I don't know).
You can just remove the lines that are not long so don't need abbreviations or leave TODOs for someone else to handle.

Reply:
I'm going to add TODOs on these lines and merge this PR as is but please change the keys in this table if you have time, thanks you!